docs: update capabilities list to add missing capabilities (#16373) (#16382)
OpenCTI-Platform/opencti: docs/docs/administration/users.md

| &nbsp;&nbsp;&nbsp;&nbsp;`Delete investigations`         | Delete existing investigations.                                                               |
| `Access connectors`                                     | Read information in the `Data > Connectors` section.                                          |
| &nbsp;&nbsp;`Manage connector state`                    | Reset the connector state to restart ingestion from the beginning.                            |

> | `API usage with authorization headers` | Controls whether users assigned to a role are allowed to authenticate to the OpenCTI API programmatically via the Authorization HTTP header. Without this capability a user will have to use the UI to login                   |

Reviewer: ```suggestion
| `API usage with authorization headers` | Programmatically authenticate against the OpenCTI API via the Authorization HTTP header. Without this capability a user will have to use the UI to log in.                   |
```

> | &nbsp;&nbsp;`Allow token usage`                    | Allow user to manage tokens & to use a token to authenticate via API programmatically via the Authorization HTTP header.                            |

Reviewer: ```suggestion
| &nbsp;&nbsp;`Allow token usage`                    | Manage tokens & use a token to programmatically authenticate against the OpenCTI API via the Authorization HTTP header.                            |
```

> | &nbsp;&nbsp;`Allow basic auth usage`                    | Ability to use basic auth to authenticate when using OpenCTI API programmatically via the Authorization HTTP header.                    |

Reviewer: ```suggestion
| &nbsp;&nbsp;`Allow basic auth usage`                    | Use basic auth to programmatically authenticate against the OpenCTI API via the Authorization HTTP header.                    |
```

| `Manage data mappers`                                   | Create, update and delete CSV & JSON mappers.                                                 |
| `Use Playbooks`                                         | Use Playbooks (enroll an entity in a playbook).                                               |
| &nbsp;&nbsp;`Manage Playbooks`                          | Manage Playbooks.                                                                             |

> | `Access PIR`                       | Ability to view PIRs (as long as the access has been granted to you).                                   |

Reviewer: ```suggestion
| `Access PIR`                       | View PIRs (as long as the access has been granted to you).                                   |
```

> | &nbsp;&nbsp;`Create/update/delete`                          | Ability to manage PIR (including access to the PIR)                    |

Reviewer: ```suggestion
| &nbsp;&nbsp;`Create/update/delete`                          | Manage PIR (including access to the PIR).                    |
```

| `Access to admin functionalities`                       | Parent capability allowing users to only view the settings.                                   |
| &nbsp;&nbsp;`Access administration parameters`          | Access and manage overall parameters of the platform in `Settings > Parameters`.              |
| &nbsp;&nbsp;`Manage credentials`                        | Access and manage roles, groups, users, organizations and security policies.                  |

> | &nbsp;&nbsp;`Manage authentication`                        | Ability for users to define Authentication strategies, password policy & set up platform-wide authentication via 2FA when using basic auth.              |

Reviewer: ```suggestion
| &nbsp;&nbsp;`Manage authentication`                        | Define authentication strategies, the password policy & set up platform-wide authentication via 2FA when using basic auth.              |
```
